### PR TITLE
Add OpenVLA model support

### LINKED ISSUE #14739: [New Model]:Can you support the VLA series models? For example, openVLA.
### The model to consider.

https://huggingface.co/openvla/openvla-7b

The vllm documentation does not support the VLA series large models. Can you support the VLA series models?

### The closest model vllm already supports.

VLM  series models

### What's your difficulty of supporting the model you want?

_No response_

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

### PR DESCRIPTION
**Pull request by yongming-qin**
## Purpose

Add support for OpenVLA model in vLLM. OpenVLA is a vision-language-action model that uses timm-based vision backbones (Prismatic architecture) with LLM backbones for action prediction tasks. This implementation follows the same pattern as DeepSeek-VL2, which also uses timm for vision processing.

This PR adds:
- Model executor implementation (`vllm/model_executor/models/openvla.py`)
- Configuration class (`vllm/transformers_utils/configs/openvla.py`)
- Processor class (`vllm/transformers_utils/processors/openvla.py`)

The implementation supports:
- Single and fused vision backbones using timm ViT models
- Multiple LLM backbones (Llama-2, Mistral, Phi-3)
- Image embedding insertion via prompt updates
- Tensor parallelism support for vision backbone

FIX #14739

## Test Plan

1. **Basic inference test:**
   ```bash
   vllm serve openvla/openvla-7b --trust-remote-code
   ```

2. **Test with image input:**
   - Use OpenAI-compatible API to send requests with image data
   - Verify image embeddings are correctly processed and inserted

3. **Compare outputs with HuggingFace implementation:**
   - Run inference on same inputs with both vLLM and HF implementations
   - Verify output logits/tokens match

## Test Result

*[To be filled after testing]*

### CODE REVIEW

**Comment by yongming-qin** [comment]
Note: Currenly the model can be loaded by vllm and we can use it to process image + text instruction. However, the results of vllm and Transformers are different. Comments and collaboration are welcome.

**Comment by hmellor** [comment]
Closing as superseded by #42654, which added OpenVLA support and merged on 2026-05-19.
